### DISCUSSION TITLE
android升级4.2.100后偶现InstantiationException：LifecycleFragment

### DISCUSSION THREAD

**Discussion by Yellowyzzz** [Tencent/libpag#1839] [Q&A]
android13，升级4.2.100后偶现，在activity初始化时报错：Unable to instantiate fragment org.extra.tools.LifecycleFragment: make sure class name exists, is public, and has an empty constructor that is public
请问有遇到过这种情况么

**Comment by CodeJhF** [comment]
抱歉这条当时没有被跟进到，是我们漏了。

因长期无回复，且 libpag 4.3 及更早版本已停止维护（当前仅维护 4.4 / 4.5），先关闭本条。
上报版本：4.2.100。

如果问题在最新版本（4.4 / 4.5）仍能复现，欢迎在本帖回复或新开 issue，并附上 libpag 版本号、平台与系统版本、可复现的 PAG 文件或最小 demo，我们会优先跟进。（stale-close）
